test/e2e/cypress/e2e/saptune_details.cy.js

```diff
@@ -0,0 +1,116 @@
+import { saptuneDetailsData } from '../fixtures/saptune-details/saptune_details_data';
+
+describe('Saptune Details page', () => {
+  const { hostname, hostID, packageVersion, configuredVersion } =
+    saptuneDetailsData;
+
+  const notFoundContainerSelector = '.pb-24';
+  const versionContainerSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(3)';
+  const saptuneServiceStatusSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(5)';
+  const saptuneTuningSolutionsSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(7)';
+  const saptuneTuningNotesSelector =
+    '.max-w-7xl > :nth-child(1) > :nth-child(1) > :nth-child(9)';
+  const saptuneStagingStatusSelector = ':nth-child(11)';
+
+  before(() => {
+    cy.visit(`hosts/${hostID}/saptune`);
+  });
+
+  it('should render saptune details not found if saptune payload version is unsupported', () => {
+    cy.loadScenario(`host-${hostname}-saptune-unsupported`);
+    cy.get(notFoundContainerSelector).should(
+      'contain',
+      'Saptune Details Not Found'
+    );
+  });
+
+  it('should render default not found if saptune payload is not installed', () => {
+    cy.loadScenario(`host-${hostname}-saptune-uninstalled`);
+    cy.get(notFoundContainerSelector).should(
+      'contain',
+      'Saptune Details Not Found'
+    );
+  });
+
+  it('should render versions list view when saptune is not tuning correctly', () => {
+    cy.loadScenario(`host-${hostname}-saptune-not-tuned`);
+
+    cy.get(versionContainerSelector)
+      .contains('Package')
+      .next()
+      .should('contain', packageVersion);
+
+    cy.get(versionContainerSelector)
+      .contains('Configured Version')
+      .next()
+      .should('contain', configuredVersion);
+
+    cy.get(versionContainerSelector)
+      .contains('Tuning')
+      .next()
+      .should('contain', 'No tuning');
+  });
+
+  it('should render each part of saptune service status with green passing icons', () => {
```

Comment:
maybe better `... services status with correct content` as we don't check the icons anymore